spaceship/views.py

```diff
@@ -302,6 +270,54 @@ def crew(team_id):
                          invite=invite,
                          achievements=achievements.for_team(team))
 
+@app.route('/invite/<team_id>', methods=['POST'])
+def invite(team_id):
+  if not current_user:
+    return jsonify({'error': 'Must be logged in.'})
+  team = get_team_if_member(team_id)
+  if not team:
+    return jsonify({'error': 'Must be a member of team.'})
+  is_captain = team.captain_id == current_user.id
+  if not is_captain:
```

Comment:
Yeah we should find a way to relax this while avoiding spamming people. If Igor invites us all to his team, and then I join and invite you and Stacey again, it would be spammy. Ideally we should only email people about joining a team once, or once in every so often... probably needs a bit more thought.

Reply:
meta-comment: we should handle stuff like this by moving forward with the current approach and leaving an issue for the problem

in this case, i would allow spammy behavior, but leave an issue like "prevent additional invites from being sent if invites have already been sent for the same people/teams", and then this would become a well-targeted issue for one of the other contributors

Reply:
I filed issue #88 to avoid spamming people. I couldn't parse from your reply whether you want to leave captaincy as a restriction or not, so I erred on the side of removing it and made it so anyone can invite people.